modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java

```diff
@@ -895,7 +900,7 @@ private DDMFormLayout _getDDMFormLayout(
 	private DDMFormValues _getDDMFormValues(
 		DDMForm ddmForm, ObjectEntry objectEntry) {
 
-		Map<String, Object> values = objectEntry.getProperties();
+		Map<String, Object> values = _getObjectEntryFieldValues(objectEntry);
```

Comment:
Note: discuss the name
`Map<String, Object> objectEntryValues = _getObjectEntryValues(objectEntry);`
or
`Map<String, Object> values = _getObjectEntryValues(objectEntry);`